package.json

```diff
@@ -35,6 +35,7 @@
     "@typescript-eslint/parser": "^1.6.0",
     "babel-eslint": "^10.0.1",
     "cbor": "^4.1.1",
+    "command-line-args": "^5.1.1",
```

Comment:
We're trying to move dependencies out of the top-level package.json and into their respective packages. Can we move this one to EVM?